app/controllers/api_controller.rb

```diff
@@ -120,7 +120,7 @@ def set_access_control_headers
   # not have these. They would instead dealing with the /api/auth
   # mechanism.
   #
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
+  if respond_to?(:verify_authenticity_token)
```

Comment:
@jrafanie Thanks! That makes sense, and I agree it is better :thumbsup;

LGTM then as far as the API is concerned